app/src/main/java/com/example/gabriel/sociala/editFeedbackActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.example.gabriel.sociala;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class editFeedbackActivity extends AppCompatActivity {
+
+    ImageView postImageView;
+    Button backButton, nextButton;
+    Button filterButton, tagButton, stickerButton, musicButton;
+    ImageView profilePic;
+    TextView userName;
+    EditText caption, reason;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit_feedback);
+
+        postImageView = findViewById(R.id.image_view_photo);
+        backButton = findViewById(R.id.back_button);
+        nextButton = findViewById(R.id.next_button);
+        filterButton = findViewById(R.id.filter_button);
+        tagButton = findViewById(R.id.tag_button);
+        stickerButton = findViewById(R.id.sticker_button);
+        musicButton = findViewById(R.id.music_button);
+
+        userName = findViewById(R.id.textView_username);
+        profilePic = findViewById(R.id.profile_image);
+
+        caption = findViewById(R.id.editText_caption);
+        reason = findViewById(R.id.editText_reason);
+
+        backButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // TODO: back to previous page.
+                finish();
+            }
+        });
+
+        nextButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                finish();
```

Comment:
would this go to PostFeedbackActivity?

Reply:
I might have mistaken the workflow. Should it be: home -> click on one of the posts -> if you have access then the “post feedback” button will show up -> click the button and goes to edit feedback -> click the “next” arrow and add video of the edit process and then hit post. Please correct me if I made any mistakes.

Reply:
Therefore as from my understanding, the back arrow button would go back to the view post page?

Reply:
What's postfeedback? Isnt that after you edited the photos/captions and provided feedback??? And then this is like the preview of the feedback. 

So you are right that it's post -> detail post, which has provide feedback button -> goes to edit feedback (thru that button) -> postFeedback -> detail post 

Or maybe I miss read the postFeedback.... But it seems to be like the review of your feedback and making sure you want to write this.

Reply:
yeah I might have developed the figma workflow directly from our paper prototype, in which we have these exact two views:
<img width="553" alt="Screen Shot 2019-11-26 at 01 46 19" src="https://user-images.githubusercontent.com/29551378/69618377-a7331f80-0fee-11ea-9736-56e5524adcd3.png">
EditFeedbackActivity is the first and PostFeedbackActivity is the second. We can definitely combine these two into just one view if we all think it is better this way.

Reply:
I personally like this flow, because it gives the user the ability to see their changes and how the feedback will look like.